### PR TITLE
Invites empty

### LINKED ISSUE #164: Fix Modal to display warning when a user has no invites
When user has no invites and they click the show invites button, the modal should display "You have no invites :("

### PR DESCRIPTION
**Pull request by chloeprezelski**
When "Show Invites" button is pressed and the user has no invites, the modal should appear and indicate that the user has no invitees. Previously, the show invites button would not do anything if the user did not have any invites.

UI updates is as follows:
image

LMK if anyone has suggestions for UI improvements.

### CODE REVIEW

File: mobile/src/pods/PodAnalytics.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {
   VictoryPie,
   VictoryLabel,
   VictoryAxis,
+  VictoryLegend,
```
Comment (allysmith12) [inline mobile/src/pods/PodAnalytics.tsx]: are all the PodAnalytics changes also supposed to be in here?

File: mobile/src/pods/PodsHomeScreen.tsx [hunk 2]
```diff
@@ -140,6 +142,7 @@ const PodsHomeScreen: React.FC<Props> = ({ navigation }) => {
       <Button
         title="Show Invites"
         onPress={() => {
+          console.log("invites button pressed");
```
Comment (allysmith12) [inline mobile/src/pods/PodsHomeScreen.tsx]: remove console

File: mobile/src/pods/PodInviteModal.tsx [hunk 3]
```diff
@@ -37,55 +41,87 @@ const PodInviteModal: React.FC<Props> = ({
...
+              }}
+            >
+              <Text style={{ color: "#5A67D8", textAlign: "center" }}>
+                Hide Modal
+              </Text>
+            </TouchableHighlight>
+          </View>
+        ) : (
+          <View style={styles.modalView}>
+            <View
```
Comment (allysmith12) [inline mobile/src/pods/PodInviteModal.tsx:100]: Could you reuse the acceptRejectButtonContainer style here?

**Review by allysmith12 — COMMENTED** [review]
Let me know what you think about my thoughts and then I will approve!

File: mobile/src/pods/PodAnalytics.tsx (hunk 1 above)
Comment (chloeprezelski) [reply mobile/src/pods/PodAnalytics.tsx]: no lol i think i messed up the branches lemme try and fix

File: mobile/src/pods/PodInviteModal.tsx (hunk 3 above)
Comment (chloeprezelski) [reply mobile/src/pods/PodInviteModal.tsx:100]: If you mean for the View below modal view, no because we need to center the text.

**Review by allysmith12 — APPROVED** [review]

**Pull request merged**